app/assets/javascripts/discourse/controllers/review-index.js.es6

```diff
@@ -127,6 +168,27 @@ export default Controller.extend({
       this.send("refreshRoute");
     },
 
+    resetFilters() {
+      this.setProperties({
+        type: null,
+        priority: "low",
+        status: "pending",
+        category_id: null,
+        username: "",
+        from_date: null,
+        to_date: null,
+        sort_order: "priority",
+        additional_filters: null
+      });
+
+      this.send("refreshRoute");
+    },
+
+    setFilters(new_filters) {
+      this.setProperties(JSON.parse(new_filters));
```

Comment:
This is fairly jank - how do I filter this to approved properties and ideally accept an object from Handlebars?